modules/ROOT/pages/get-mapping-recommendations.adoc

```diff
@@ -19,13 +19,13 @@ This view shows the expected structures of the input and output messages. Mappin
 
 image::simple-transform-card-example.png[]
 
-If you are not sure how to map an input payload to an output payload, you can have the Transform card recommend mappings between them.
+If you are not sure how to map an input data type to an output type, you can have the Transform card recommend mappings between them.
 
 == Limitations
 
 * The card does not make recommendations for mapping input attributes to output attributes or input variables to output variables.
```

Comment:
`input` and `output`  ? I won't keep commenting because I'm not even sure it's required.

Reply:
Not required. Those are not names.